lib/rules/listitem.json

```diff
@@ -5,7 +5,7 @@
 	"tags": ["cat.structure", "wcag2a", "wcag131"],
 	"metadata": {
 		"description": "Ensures <li> elements are used semantically",
-		"help": "<li> elements must be contained in a <ul> or <ol>"
+		"help": "<li> elements must be contained in a <ul> or <ol> without a role or an element with role=list."
```

Comment:
Can you swap around the description and help text. The help text needs to be shorter than the description. Also, I think this needs to be done in the remediation message of the check. Preferably, this is put in conditionally. If it's not in a UL, it just has the message it has now, if it is in a UL with a role, the message add something like `without a role, or with role="list"`.